modules/querying/pages/func/type-conversion-functions.adoc

```diff
@@ -186,9 +186,11 @@ Converts a number to a string using the C++ printf function with the `g` specifi
 === Example
 
 ----
-to_string(1000.1) -> "1000.1"
-to_string(0.0000000001) -> "1E-10"
-to_string(12300000000) -> "1.23E7"
+to_string(1000.1) -> "1000.1"  
+to_string(0.0000000001) -> "1E-10"  
+to_string(12300000000::INT) -> "12300000000"  
+to_string(12300000000::FLOAT) -> "1.23e+10"  
+to_string(12300000000::DOUBLE) -> "1.23e+10"  
```

Comment:
So we support this exact syntax? 
I guess this is some C++ way of specifying data type that I am unfamiliar with. Is is casting the number to a target type?